packages/solana/src/deposit.ts

```diff
@@ -49,14 +49,15 @@ const getTransferAmount = (
 
 export const findSolanaDepositSignature = async (
   solanaEndpoint: string,
+  signatureAddress: string,
```

Comment:
the signature address (vault swap program) is different from the deposit address (token vault pda) for vault swaps